Assets/ObstacleTower/Scripts/AgentLogic/ObstacleTowerManager.cs

```diff
@@ -77,7 +77,7 @@ private void UpdateEnvironmentParameters()
             Debug.Log("visual-theme outside of valid range. Using default value.");
         }
         
-        var perspective = (int)Academy.Instance.EnvironmentParameters.GetWithDefault("perspective", (int)AgentPerspective.ThirdPerson);
+        var perspective = (int)Academy.Instance.EnvironmentParameters.GetWithDefault("agent-perspective", (int)AgentPerspective.ThirdPerson);
```

Comment:
Is there a reason for renaming this environment parameter? Should the documentation change?

Reply:
This fixes a typo which I think was introduced when you moved the code to the new environment parameter system.

Reply:
Arf. Good catch then.